### ISSUE TITLE
Video Task Dataset Summary

### ISSUE THREAD

**Issue by huuuuusy** [huuuuusy/CV-Learning-Everyday#3]
Video tasks mainly have two types of dataset, trimmed videos and unedited videos. The  trimmed video contains a clear action and the label is unique, while the unedited video always contains a lot of useless information.

**Comment by huuuuusy** [comment]
|Dataset|Paper|Info|
|:--: |:--: |:--: |
|[HMDB-51](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/)|【ICCV2011】[HMDB: A large video database for human motion recognition.](https://dspace.mit.edu/handle/1721.1/69981)|51 categories, 6,766 clip videos, each video no more than 10 seconds, resolution 320 * 240, a total of 2 GB. The video comes from YouTube and Google Video, and the content includes the faces of people's faces, limbs, and objects.|
|[UCF-101](https://www.crcv.ucf.edu/data/UCF101.php)|【CoRR2012】[UCF101: A dataset of 101 human action classes from videos in the wild.](https://arxiv.org/abs/1212.0402)|101 categories, 13,320 video clips, each video no more than 10 seconds, a total of 27 hours, resolution 320 * 240, a total of 6.5 GB. The video is based on YouTube and includes five categories: makeup brushing, crawling, haircutting, playing instruments, and sports.|
|[Sports-1M](https://cs.stanford.edu/people/karpathy/deepvideo/classes.html)|【CVPR2014】[Large-scale video classification with convolutional neural networks.](http://vision.stanford.edu/pdf/karpathy14.pdf)|487 categories, 1,100,000 videos (70% training, 20% verification, 10% testing). The content includes a variety of sports.|
|[Charades](https://allenai.org/plato/charades/)|【ECCV2016】[Hollywood in Homes: Crowdsourcing Data Collection for Activity Understanding](https://arxiv.org/abs/1604.01753)|57 categories, 9,848 unedited videos (7,985 training, 1,863 tests), approximately 30 seconds per video. Each video has multiple tags with the start and end time of each action.|
|[ActivityNet](http://activity-net.org/)|【CVPR2015】[A large-scale video benchmark for human activity understanding.](https://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Heilbron_ActivityNet_A_Large-Scale_2015_CVPR_paper.pdf)|200 categories, 19,994 unedited video (10,024 training, 4,926 verification, 5,044 test), a total of 648 hours. The content includes diet, exercise, family activities and more.|
|[Kinetics](https://deepmind.com/research/open-source/open-source-datasets/kinetics/)|【CoRR2017】[The Kinetics human action video dataset](https://arxiv.org/pdf/1705.06950.pdf)|400 class, 246k training video, 20k verification video, each video is about 10 seconds. The video is from YouTube. Kinetics is a large-scale dataset whose role in video understanding area is similar to ImageNet in image recognition, and some work migrates to other video datasets using the Kinetics pre-training model.|
|[YouTube-8M](https://research.google.com/youtube8m/)|【CoRR2016】[YouTube-8M: A large-scale video classification benchmark](https://arxiv.org/pdf/1609.08675.pdf)|4716 class, 7M video, a total of 450,000 hours. It is difficult to download or train.|
|[Something-something](https://20bn.com/datasets/something-something)|【ICCV2017】[The "something something" video database for learning and evaluating visual common sense](https://arxiv.org/abs/1706.04261)174 classes, 108,000 videos, 2 to 6 seconds per video. Unlike Kinetics, the Something-something dataset requires a more granular, more underlying interaction, such as "push from left to right" and "push from right to left."|
